changelog/10502.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed tmp_path_retention_policy crash when skipping from fixture.
```

Comment:
No changelog needed I think, as the original change hasn't been released yet.

Reply:
Removed.